modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// Configuring Bitbucket server OAuth1
+
+
+[id="proc_configuring-bitbucket-server-oauth1_{context}"]
+= Configuring Bitbucket server OAuth1
```

Comment:
Configuring Bitbucket server OAuth1

Is the `1` suffix at the end of the title and file a typo?

Reply:
no. that was made intentionally.

Reply:
OK